src/azul/service/source_service.py

```diff
@@ -94,9 +97,14 @@ def _dynamodb(self):
         return aws.dynamodb
 
     def _get(self, key: str) -> list[AnyJSON]:
+        log.debug('_request(GET, DynamoDB, key=%r) …', key)
```

Comment:
Maybe instead of or in addition to the literal string "DynamoDB" we could log the table name or ARN?